### PR TITLE
Rfairooz/socials calendar

### LINKED ISSUE #160: Limit Events to 8 count
- Currently the Events in the upcoming section will display ALL upcoming events (right now it is only 8 but more will be added and the problem will not scale very nicely)
- We want to only show the first 8 events 
- Modify the array to show only the first 8 events (you can add a test event for now and remove it later on)
- Check out the upcoming section on the home page to see the implementation

### LINKED ISSUE #167: Add Socials to Events
- Contact Joshua C and Alex H for events/times/places for the socials and put them into the events
- Ensure you follow the event structure

### LINKED ISSUE #200: Color Code Upcoming Events on Home Page
- Change the box shadow colors to their respective colors based on the filters provided in the events page

### LINKED ISSUE #201: Color Code Upcoming Events on Events Page
- Color code based on filters on calendar in events page

### LINKED ISSUE #209: Check Code Logic for Missing `0` before 9 in Date
- Looking at the other dates, if the number is < 10 then the number has a leading 0
- When 9 is used, it does not have the leading 0 when it should 
- Check out the logic use to figure out if a leading zero is needed and fix it to put the 0 in front of the 9

### LINKED ISSUE #210: Make Calendar Arrows Larger
- Make the Calendar Arrows larger when on a mobile screen

### PR DESCRIPTION
**Pull request by shahdivyank**
Helloo, I linked all the issues. The color coding issue was actually linked to the home page so I talked to Kavin about that. I was already on it so I just finished it. I could not figure out how to read in the color with tailwind css. I know the syntax but it is not reading in the color so right now I have pure css for that. Waiting for your feedback on that. Let me know!
Screen Shot 2022-09-09 at 10 33 42 PM
Screen Shot 2022-09-09 at 10 33 59 PM
Screen Shot 2022-09-09 at 10 34 10 PM

### CODE REVIEW

File: components/Events.js [hunk 1]
```diff
@@ -40,6 +49,15 @@ export const Events = [
 		end: new Date(2022, 9, 13, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Mentor Q/A",
+		location: "ACM Clubroom",
```
Comment (shahdivyank) [inline components/Events.js]: WCH 127

**Comment by shahdivyank** [comment]
Otherwise everything looks good! Dw bout the color issue, the current solution is also good. I ran into that same issue when I was building the components so it looks like this is the only way :(

**Comment by rfairooz** [comment]
> Otherwise everything looks good! Dw bout the color issue, the current solution is also good. I ran into that same issue when I was building the components so it looks like this is the only way :(

ahh ok ok :( thank u!

File: components/Events.js (hunk 1 above)
Comment (rfairooz) [reply components/Events.js]: I'll change that. Btw, Josh did say those are potential locations not final ones but final ones will take them weeks to know if they change. This is alright for now?

Comment (shahdivyank) [reply components/Events.js]: Yea thats fine!

**Review by rfairooz — APPROVED** [review]

**Review by skyguy-netizen — APPROVED** [review]

**Pull request merged**